template/runner-config.tpl

```diff
@@ -17,22 +17,17 @@ listen_address = "${prometheus_listen_address}"
   request_concurrency = ${runners_request_concurrency}
   output_limit = ${runners_output_limit}
   limit = ${runners_limit}
-  [runners.docker]
-    tls_verify = false
-    image = "${runners_image}"
-    privileged = ${runners_privileged}
-    disable_cache = ${runners_disable_cache}
-    volumes = ["/cache"${runners_additional_volumes}]
-    extra_hosts = ${jsonencode(runners_extra_hosts)}
-    shm_size = ${runners_shm_size}
-    pull_policy = ${runners_pull_policies}
-    runtime = "${runners_docker_runtime}"
-    helper_image = "${runners_helper_image}"
-    ${runners_docker_services}
+
+  ${runners_docker_options}
```

Comment:
**suggestion:** since the template is now only four lines, WDYT about just putting it inline here? Then we can remove the template, the local variable and directly feed it as a variable to the `runner-config.tpl`.

Reply:
It's probably good practice to keep it as a template in case it grows again.

Reply:
I stay with the template for now.